docs/guides/update-guide/820-to-830.md

```diff
@@ -10,6 +10,16 @@ The following sections explain which adjustments must be made to migrate from Ca
 
 ## Zeebe
 
+:::caution
+When upgrading to 8.3, Zeebe runs a migration. Depending on the amount of active element instances in the cluster, this
+migration may cause a longer startup time than usual. While this migration is running, users may experience:
+
+- `DEADLINE_EXCEEDED` messages in job workers.
+- Unhealthy partitions in the Zeebe cluster.
+  - For SaaS users, this marks the entire Zeebe cluster as unhealthy.
+- No processing happens during the migration.
+  :::
```

Comment:
We need to remove the extra spaces for the caution block to work
```suggestion
:::
```

Reply:
Good catch 👍 Guess I shouldn't auto format the documentation 😅